Resolve syncing conflicts from repo_sync_working_branch to public (#446)
MicrosoftDocs/Viva: Viva/insights/advanced/analyst/templates/business-resilience.md

---
title: Business resilience report

> description: Learn how to use the Microsoft Viva Insights Power BI template to to compare employee behavior before and after a business transition

Reviewer: ```suggestion
description: Learn how to use the Microsoft Viva Insights Power BI template to compare employee behavior before and after a business transition
```

> >[!Note]

Reviewer: ```suggestion
> [!NOTE]
```

This page shows how employee collaboration patterns are changing in response to a shift in work patterns, and which collaboration tools people are substituting for in-person interactions. 

Transitions at work can cause changes in employee collaboration patterns as they adjust to a new normal. Groups with large increases in collaboration might be facing challenges managing their workload, while groups with significantly reduced workloads might be struggling to adapt.

### Manager coaching

**How has manager coaching changed over time?**

This page shows how manager coaching has been impacted by your business’s recent transition.

> Managers are crucial factors in the development, engagement, and wellbeing of their reports and in periods of transition, managers help their reports prioritize and provide stability. However, long periods of transition could also require managers to take on new or additional responsibilities, leaving less time for them to support their teams. We recommend that individual contributors spend a minimum of one hour per month in 1:1 meetings with their managers, or 15 minutes per week on average. Employees with this level of 1:1 interaction with their managers are called *coached employees* in this report.

Reviewer: ```suggestion
Managers are crucial factors in the development, engagement, and well-being of their reports, and in periods of transition, managers help their reports prioritize and provide stability. However, long periods of transition could also require managers to take on new or additional responsibilities, leaving less time for them to support their teams. We recommend that individual contributors spend a minimum of one hour per month in 1:1 meetings with their managers, or 15 minutes per week on average. Employees with this level of 1:1 interaction with their managers are called *coached employees* in this report.
```
